1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,31 @@
+import { fizzbuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
+  it.each([3, 9, 42])("should return Fizz when given %i", (n) => {
```

Comment:
Overall looks good.
I like the `%i` for describing each example with the iteration.
Great job on the red-green-refactor sequence 👍🏼